docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_WideAndDeep_Family_Models.md

```diff
@@ -29,6 +29,7 @@ by changing the `build_model_columns()` function in `census_dataset.py` as follo
 
 ```python
 def build_model_columns():
+  import tensorflow as tf
```

Comment:
why? it is already imported in the original repo

Reply:
It was added during checks and can be removed.

Reply:
Removed.